electricitymap/contrib/lib/models/event_lists.py

```diff
@@ -39,7 +39,9 @@ def append(self, **kwargs):
         pass
 
     def to_list(self) -> List[Dict[str, Any]]:
-        return [event.to_dict() for event in self.events]
+        event_list = [event.to_dict() for event in self.events]
+        event_list.sort(key=lambda x: x["datetime"])
+        return event_list
```

Comment:
Nice improvment but can we chain it directly to avoid an object assignment?

Reply:
hhm you mean `return sorted([event.to_dict() for event in self.events], key=lambda x: x["datetime"])` ?

Reply:
Yep. If that makes sense?

Reply:
Just updated :)